### PR TITLE
feat: add external-sites app

### PR DESCRIPTION
**Pull request by JammingBen**
See README.md file.

refs https://github.com/owncloud/web/issues/7260

### CODE REVIEW

File: packages/web-app-external-sites/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+      "typeRoots" : ["./node_modules/@types"]
```
Comment (dschmidt) [inline packages/web-app-external-sites/tsconfig.json:4]: Most probaly not needed

File: packages/web-app-external-sites/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      "target": "external",
+      color: "#0D856F",
+      "icon": "cloud",
+      "priority": 50
+    }
+  ]
+}
+```
+
+The `name` will appear in the app menu, while `url` specifies the url which will be opened or embedded. `target` can either be `external` or `embedded`. Choose `external` if the site should be opened in a new tab or window. Choose `embedded` if the external content should be embedded within ownCloud Web. All of these 3 config options are required.
```
Comment (kulmann) [inline packages/web-app-external-sites/README.md:24]: Could you please add a statement that `embedded` only works if the server of the target url allows embedding via their `CORS` server settings? Together with a rule of thumb `Just test it. If embedded doesn't work for the specified url then you need to use external instead.`.

File: packages/web-app-external-sites/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+The following options are optional:
```
Comment (kulmann) [inline packages/web-app-external-sites/README.md:26]:
```suggestion
The following attributes are optional:
```

File: packages/web-app-external-sites/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- `color` _(string)_ - specifies the Hex color codes of the menu item.
```
Comment (kulmann) [inline packages/web-app-external-sites/README.md:28]:
```suggestion
- `color` _(string)_ - specifies the Hex color codes of the icon background of the menu item.
```

File: packages/web-app-external-sites/README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+- `icon` _(string)_ - specifies the name of a [Remix Icon](https://remixicon.com/) to be used for the menu item.
+- `priority` _(number)_ - specifies the order of the menu item.
```
Comment (kulmann) [inline packages/web-app-external-sites/README.md:30]: Please add some words about the range and what happens if it's left out.

File: packages/web-app-external-sites/src/App.vue [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="external-site-container">
```
Comment (kulmann) [inline packages/web-app-external-sites/src/App.vue:2]: This should be a `<main>` tag for a11y reasons... we have a button to jump to the `main` content. Editor apps have that from `AppWrapper.vue` but standalone apps need to define it.

**Review by kulmann — CHANGES_REQUESTED** [review]

**Review by kulmann — APPROVED** [review]
awesome 🥳

**Pull request merged**